Streetcode/Streetcode.BLL/MediatR/Streetcode/Text/GetByStreetcodeId/GetTextByStreetcodeIdHandler.cs

```diff
@@ -34,7 +37,14 @@ public GetTextByStreetcodeIdHandler(IRepositoryWrapper repositoryWrapper, IMappe
         }
 
         var textDto = _mapper.Map<TextDTO?>(text);
+
+        if(text is null)
+        {
+            return Result.Fail(new Error(""));
```

Comment:
didn't write anything in error message, because the new error message functionality will be added soon.